src/main/java/org/loose/fis/mov/controllers/RegistrationController.java

```diff
@@ -102,4 +116,19 @@ private void checkFieldsForNull() throws EmptyFieldException {
             throw new EmptyFieldException();
         }
     }
+
+    private Stage stage;
+    private Scene scene;
+    private Parent root;
```

Comment:
Can't these variables be local to `switchToPopUp()`?

> `private Stage stage;`
> `private Scene scene;`
> `private Parent root;`

Reply:
am scapat de ala